requirements.txt

```diff
@@ -20,12 +20,14 @@ numpy==1.19.2
 pandas==1.0.3
 pyrsistent==0.16.0
 python-dotenv==0.12.0
+python-json-logger==2.0.1
 psycopg2-binary==2.8.5
 pytest==6.1.1
 pytz==2019.3
 requests==2.24.0
 scipy==1.5.0
 SQLAlchemy==1.3.18
 six==1.14.0
+slack-sdk==3.2.0
```

Comment:
Nit: since we're not using slack anymore, we can get rid of this in the reqs.txt